src/ensembl/src/shared/toggle-image-button/ToggleImageButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import dafaultStyles from './ToggleImageButton.scss';
+import ImageHolder from './ImageHolder';
+
+export enum ImageButtonStatus {
+  ACTIVE = 'active',
+  INACTIVE = 'inactive',
+  DISABLED = 'disabled'
+}
+
+type Props = {
+  buttonStatus: ImageButtonStatus;
+  description?: string;
```

Comment:
If it's an optional prop, it shouldn't be defined in defaultProps. Alternatively, if it is defined in defaultProps, then it's not an optional prop.

Reply:
That's done.